src/App.tsx

```diff
@@ -1,12 +1,62 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import { makeStyles } from "@mui/styles";
+import { Button } from "@mui/material";
+import { Box } from "@mui/system";
 
 
-function App() {
+const useStyles = makeStyles({
+  container: {
+    width: '90%',
+    margin: '50px auto',
+    textAlign: 'center'
+  },
+  h1: {
```

Comment:
Give this a more descriptive name like: Title

Reply:
Done